filters/filters.txt

```diff
@@ -77925,3 +77925,12 @@ www.youtube.com###foreground-content
 ! youtube.com
 youtube.com###banner
 youtube.com##.style-scope.ytd-action-companion-ad-renderer.no-transition
+
+!!!! DRAFT !!!!
```

Comment:
These need to be sorted in alphabetical order to match above filters, currently unable to so ill do it later.